dart/simulation/World.cpp

```diff
@@ -244,23 +244,35 @@ void World::step(bool _resetCommand)
   mConstraintSolver->setContactClippingDepth(mContactClippingDepth);
   mConstraintSolver->setFallbackConstraintForceMixingConstant(
       mFallbackConstraintForceMixingConstant);
-  runConstraintEngine(_resetCommand);
+  runRegisteredConstraintEngine(_resetCommand);
   integratePositions(initialVelocity);
 
   mTime += mTimeStep;
   mFrame++;
 }
 
 //==============================================================================
-void World::runConstraintEngine(bool _resetCommand)
+void World::runRegisteredConstraintEngine(bool _resetCommand)
 {
-  mConstraintEngineFn(_resetCommand);
+  mRegisteredConstraintEngine(_resetCommand);
 }
 
 //==============================================================================
 void World::runLcpConstraintEngine(bool _resetCommand)

```

Comment:
Does it actually make sense to have `_resetCommand` here?  Doesn't the LCP create the impulses in the first place? Why would we ever not want to reset after that?